.gitignore

```diff
@@ -1,5 +1,7 @@
+/scripts
```

Comment:
Why adding scripts to ignore list?

Reply:
i want rebase the master,and remove the extra commits

Reply:
![image](https://user-images.githubusercontent.com/6780391/34713777-329224f8-f562-11e7-90b0-434ac99b6fba.png)